publiq-ui/pub-link.vue

```diff
@@ -1,13 +1,25 @@
 <template>
-  <a :href="href" @click="handleClick"><slot /></a>
+  <component :is="as" v-bind="$attrs" v-on="$listeners" @click="handleClick">
```

Comment:
I'm curious, is `v-bind="$attrs"` different from `v-bind="$props"` and when should we use which one?
Also good idea to bind the `listeners` in this case 👍

Reply:
`v-bind="$props"` only passes the props defined in the pub-link component, `v-bind="$attrs"` passes all the props given, no matter if they've been defined as props in pub-link

Reply:
Okay thanks for explaining!